doc_admin/AdminManual.adoc

```diff
@@ -84,7 +84,7 @@ More information can be found on the Open Hospital website: https://www.open-hos
 
 NOTE: Do not double-click on the **oh.bat** startup script without unzipping the package first!
 
-Most operating systems allow to "open" a downloaded compressed archive by just double-clicking on it. 
+Most operating systems allow one to "open" a downloaded compressed archive by just double-clicking on it. 
```

Comment:
Together with @dbmalkovsky we have spent quite some time removing all the "one" reference from the manual. Are we reverting that ? :-(

Reply:
Good point!   How about: `A downloaded compressed archive can be opened on most operating systems by just double-clicking on it.`     That gets rid of the `one` reference.

Reply:
ok for me